### PR TITLE
[Configurable Autosubmit] Approval Validation uses autosubmit configurations - Part 6

### PR DESCRIPTION
**Pull request by ricardoamador**
Update the approval service so that is uses the autosubmit configurations and removes the checks on MEMBER and OWNER.

Note: this will not pass checks until the other pull requests are merged.

*List which issues are fixed by this PR. You must list at least one issue.*

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: auto_submit/lib/validations/approval.dart [hunk 1]
```diff
@@ -84,45 +107,52 @@ class Approver {
 
   /// Parses the restApi response reviews.
   ///
-  /// If author is a MEMBER or OWNER then it only requires a single review from
-  /// another MEMBER or OWNER. If the author is not a MEMBER or OWNER then it
-  /// requires two reviews from MEMBERs or OWNERS.
+  /// If author is a of the defined approval group then it only requires a
```
Comment (keyonghan) [inline auto_submit/lib/validations/approval.dart:110]:
```suggestion
  /// If author is a member of the defined approval group then it only requires a
```

File: auto_submit/lib/validations/approval.dart [hunk 2]
```diff
@@ -84,45 +107,52 @@ class Approver {
...
-    const Set<String> allowedReviewers = <String>{ORG_MEMBER, ORG_OWNER};
-    final bool authorIsMember = allowedReviewers.contains(authorAssociation);
-
-    // Author counts as 1 review so we need only 1 more.
+  /// Changes requested will supercede any approvals and the autosubmit bot will
+  /// not make any merges until the change requests are fixed.
+  Future<void> computeApproval() async {
+    _remainingReviews = repositoryConfiguration.approvingReviews;
+    // team might be more than one in the future.
+    // TODO this might be failing if the member is not a team member with exception.
```
Comment (keyonghan) [inline auto_submit/lib/validations/approval.dart:120]: Nit: use the TODO format.

**Review by keyonghan — APPROVED** [review]
LGTM with tiny nits!

File: auto_submit/lib/validations/approval.dart (hunk 2 above)
Comment (ricardoamador) [reply auto_submit/lib/validations/approval.dart]: Done.

**Pull request merged**